uspace/app/mkext4/mkext4.c

```diff
@@ -86,6 +86,7 @@ int main(int argc, char **argv)
 
 			--argc;
 			++argv;
+			continue;
```

Comment:
If this was the last argument, then *argv is now NULL and while (*argv[0] == '-') will crash. Need to add *argv != NULL *&& to the while condition.

Reply:
You're right, fixed.